### DISCUSSION TITLE
No ability to accept mission

### DISCUSSION THREAD

**Discussion by Sull12** [Khopa/dcs_liberation#2733] [Q&A] [answered]
My apologies if this has been asked and answered. I have been playing DCS Liberation and after completing mission 1 and landing back on the carrier, I quit the mission and then alt & tab back to the pass turn / take off page but I do not have the box to accept the mission or to even manually accept it. There just is no box to select anything. What am I doing wrong?
Thank you for any help you can provide.

**Comment by Starfire13** [comment]
If you can, hop on the Discord and someone can help troubleshoot.

**Reply by Sull12** [reply]
Thank you. Not too familiar with Discord...click on the Discord Link at the top of the Planning Page in Liberation?

**Reply by Starfire13** [reply]
There is also a discord link on Liberation's main page on GitHub.

**Comment by Sull12** [comment, marked as answer]
Ty. I figured it out. Operator error (no surprise). 


Sent from Yahoo Mail for iPhone


On Thursday, February 9, 2023, 6:46 PM, Starfire13 ***@***.***> wrote:




There is also a discord link on Liberation's main page on GitHub.
